contributor-growth/drafts/contributor-growth-framework/3-Keeping-contributors-engaged.md

```diff
@@ -0,0 +1,14 @@
+# Keeping contributors engaged after their first contribution
+
+To keep contributors engaged, the human element is important. Building a warm and fuzzy environment spreads like wildfire. While it may sound obvious, it's easy to forget, so make sure you:  
+  1. Thank contributors publicly and privately (tools like Orbit auto populate a thank you tweet and email)  
```

Comment:
1. Thank contributors publicly and privately (some community CRMs auto populate a thank you tweet and email for first PRs)

Reply:
This has been added